travis_ci_install.sh

```diff
@@ -0,0 +1,18 @@
+if [[ ${TRAVIS_OS_NAME} == "osx" ]]
+then
+    # download and install EDM
+    wget https://package-data.enthought.com/edm/osx_x86_64/1.3/edm_1.3.0.pkg
+    sudo installer -pkg edm_1.3.0.pkg -target /
+else
+    # set up XVFB for GUI tests
```

Comment:
Since we are using a separate python runtime it is good practice to deactivate the default virtual environment setup on linux build nodes.